XcodeServerSDK/Utils/HTTPUtils.swift

```diff
@@ -30,15 +30,13 @@ public class HTTP {
             //try to cast into HTTP response
             if let httpResponse = response as? NSHTTPURLResponse {
                 
-                if error != nil {
-                    
+                guard error != nil else {
                     //error in the networking stack
                     completion(response: httpResponse, body: nil, error: error)
                     return
                 }
                 
-                if data == nil {
-                    
+                guard data == nil else {
```

Comment:
Here, on the other hand, we want to unwrap `data` and return if it's nil. So here I'd change it to `guard let data = data else {`. So that if `data` gets unwrapped successfully, the rest of the method will be able to use it unwrapped already.